src/ssh/test-sshbridge.c

```diff
@@ -747,7 +750,11 @@ static const TestFixture fixture_knownhost_sssd_badkey = {
   .knownhosts_sssd = "ssh-rsa Ym9ndXMK",
   .knownhosts_sssd_host = "127.0.0.1",
   .knownhosts_file = "/dev/null",
+#ifdef HAVE_SSH_SESSION_HAS_KNOWN_HOSTS_ENTRY
+  .problem = "unknown-host"
+#else
```

Comment:
See previous comment. Not entirely sure what to do in this situation as libssh api recognizes it as unknown because it's invalid while the cockpit api gets through the pre-connection check.